### PR TITLE
Add other estimators 

### PR DESCRIPTION
**Pull request by zackt113**

### CODE REVIEW

File: src/strapvizpy/display.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from strapvizpy.bootstrap import calculate_boot_stats
 
 
-def plot_ci(sample, rep, bin_size=30, n="auto", ci_level=0.95,
+def plot_ci(sample, rep, bin_size=30, ci_estimator="mean", n="auto", ci_level=0.95,
             ci_random_seed=None, title="", x_axis="Bootstrap Sample Mean", 
             y_axis="Count", path=None):
```
Comment (gauthampughazhendhi) [inline src/strapvizpy/display.py]: @zackt113 can you rename all the ci_* variables like estimator, level, random_seed in all the places including documentation?

File: src/strapvizpy/display.py [hunk 2]
```diff
@@ -72,20 +74,24 @@ def plot_ci(sample, rep, bin_size=30, n="auto", ci_level=0.95,
...
         
     plt.hist(sample_stat_dict[1], density=False, bins=bin_size)
     plt.axvline(sample_stat_dict[0]["lower"], color='k', linestyle='--')
-    plt.axvline(sample_stat_dict[0]["sample_mean"], color='r', linestyle='-')
+    plt.axvline(sample_stat_dict[0]["sample_"+ci_estimator], color='r', linestyle='-')
     plt.axvline(sample_stat_dict[0]["upper"], color='k', linestyle='--')
     axes = plt.gca()
     _, y_max = axes.get_ylim()
-    plt.text(sample_stat_dict[0]["sample_mean"], 
+    plt.text(sample_stat_dict[0]["sample_"+ci_estimator],
```
Comment (gauthampughazhendhi) [inline src/strapvizpy/display.py]: Also can you update the plot's x axis label to "Bootstrap sample" + estimator

**Review by gauthampughazhendhi — CHANGES_REQUESTED** [review]
@zackt113 please find my changes

**Comment by zackt113** [comment]
@gauthampughazhendhi, all changes have been made, let me know how it looks. Thanks!

File: src/strapvizpy/display.py [hunk 3]
```diff
@@ -21,17 +20,17 @@ def plot_ci(sample, rep, bin_size=30, n="auto", ci_level=0.95,
     bin_size = int
         a number of bins representing intervals of equal size
         over the range
+    estimator: {"mean", "median", "var", "sd"}
```
Comment (voremargot) [inline src/strapvizpy/display.py:23]: Add in that the default is the mean 
```suggestion
    estimator: {"mean", "median", "var", "sd"}, default = "mean"
```

**Review by voremargot — COMMENTED** [review]
I have just added in the default for the new estimator parameter! Looks good!

**Comment by gauthampughazhendhi** [comment]
@zackt113 Looks good to me

**Review by gauthampughazhendhi — APPROVED** [review]
Looks good @zackt113

**Pull request merged**